src/pages/test-report/cat-a-mod2/test-report.cat-a-mod2.page.ts

```diff
@@ -196,16 +173,13 @@ export class TestReportCatAMod2Page extends BasePageComponent {
       isRemoveFaultMode$.pipe(map(result => (this.isRemoveFaultMode = result))),
       isSeriousMode$.pipe(map(result => (this.isSeriousMode = result))),
       isDangerousMode$.pipe(map(result => (this.isDangerousMode = result))),
-      manoeuvres$.pipe(map(result => (this.manoeuvresCompleted = result))),
       testData$.pipe(
         map((data) => {
-
-          // TODO - PREP-AMOD2: Use a mod2 test category
           this.isTestReportValid =
-            this.testReportValidatorProvider.isTestReportValid(data, TestCategory.BE);
+            this.testReportValidatorProvider.isTestReportValid(data, TestCategory.EUAM2);
           this.missingLegalRequirements =
-            this.testReportValidatorProvider.getMissingLegalRequirements(data, TestCategory.BE);
-          this.isEtaValid = this.testReportValidatorProvider.isETAValid(data, TestCategory.BE);
+            this.testReportValidatorProvider.getMissingLegalRequirements(data, TestCategory.EUAM2);
+          this.isEtaValid = this.testReportValidatorProvider.isETAValid(data, TestCategory.EUAM2);
```

Comment:
use category from store to pass through the testReportValidatorProvider instead of hardcoded category

Reply:
updated to access the test cat from the state.